docs/Using-Fleet/configuration-files/README.md

```diff
@@ -290,12 +290,10 @@ integrations
 webhook_settings
 ```
 
-You can bypass these errors by removing the key from your YAML or adding the `--force` flag. This flag will force application of the changes without validation. Proceed with caution.
+You can bypass these errors by removing the key from your YAML or adding the `--force` flag. This flag will force the application of the changes without validation but should be used with caution.
```

Comment:
Tiny thing but I'd use "and" instead of "but" here. Also, I think we can chop some more words 🔪 😇 
```suggestion
You can bypass these errors by removing the key from your YAML or adding the `--force` flag. This flag will apply the changes without validation and should be used with caution.
```